src/analysis/retail/monk/mistweaver/modules/core/AplChoiceDescription.tsx

```diff
@@ -0,0 +1,122 @@
+import SPELLS from 'common/SPELLS';
+import talents from 'common/TALENTS/monk';
+import { AlertWarning, SpellLink } from 'interface';
+import { useInfo } from 'interface/guide';
+import { MistweaverApl } from './AplCheck';
+
+const aplTitle = (choice: MistweaverApl) => {
+  switch (choice) {
+    case MistweaverApl.RisingMistAncientTeachings:
+      return (
+        <>
+          <SpellLink id={talents.RISING_MIST_TALENT} /> /{' '}
+          <SpellLink id={talents.ANCIENT_TEACHINGS_TALENT} />
+        </>
+      );
+    default:
+      return <em>Fallback</em>;
+  }
+};
+
+const RisingMistDescription = () => {
+  return (
+    <>
+      <SpellLink id={talents.RISING_SUN_KICK_TALENT} /> to extend hots and{' '}
+    </>
+  );
+};
+
+const AncientTeachingsDescription = () => {
+  const info = useInfo();
+  return (
+    <>
+      <SpellLink id={talents.ESSENCE_FONT_TALENT} />{' '}
+      {info?.combatant.hasTalent(talents.FAELINE_STOMP_TALENT) ? (
+        <>
+          or <SpellLink id={talents.FAELINE_STOMP_TALENT} />{' '}
+        </>
+      ) : (
+        <></>
+      )}
+      to empower your damaging abilities <SpellLink id={talents.RISING_SUN_KICK_TALENT} />,{' '}
+      <SpellLink id={SPELLS.BLACKOUT_KICK} />, and <SpellLink id={SPELLS.TIGER_PALM} /> to heal via{' '}
+      <SpellLink id={talents.ANCIENT_TEACHINGS_TALENT} />.
```

Comment:
I think something like `to heal by using your damaging abilities (TP, RSK, BOK) via Ancient Teachings` would sound better

Reply:
![image](https://user-images.githubusercontent.com/26779541/219973039-29edbb2a-d994-4b19-a6dd-8b3980f14dce.png)